packages/components/src/test/data/users.ts

```diff
@@ -186,9 +189,44 @@ export const TEST_USER_APP_ADMIN = new User({
         PermissionTypes.Insert,
         PermissionTypes.Update,
         PermissionTypes.ManagePicklists,
+        PermissionTypes.ManageSampleWorkflows,
         PermissionTypes.Admin,
         PermissionTypes.UserManagement,
         PermissionTypes.ApplicationAdmin,
         PermissionTypes.AddUser,
     ]),
 });
+
+export const TEST_USER_STORAGE_DESIGNER = new User({
+    canDelete: false,
+    canDeleteOwn: false,
+    canInsert: false,
+    canUpdate: false,
+    canUpdateOwn: false,
+    isAdmin: false,
+    isAnalyst: false,
+    isDeveloper: false,
+    isGuest: true,
+    isRootAdmin: false,
+    isSignedIn: false,
+    isSystemAdmin: false,
+    isTrusted: false,
+    permissionsList: List<string>([PermissionTypes.Read, PermissionTypes.DesignStorage]),
+});
+
+export const TEST_USER_STORAGE_EDITOR = new User({
+    canDelete: false,
+    canDeleteOwn: false,
+    canInsert: false,
+    canUpdate: false,
+    canUpdateOwn: false,
+    isAdmin: false,
+    isAnalyst: false,
+    isDeveloper: false,
+    isGuest: true,
+    isRootAdmin: false,
+    isSignedIn: false,
+    isSystemAdmin: false,
+    isTrusted: false,
+    permissionsList: List<string>([PermissionTypes.Read, PermissionTypes.EditStorageData]),
```

Comment:
and picklist and workflow permissions?

Reply:
Yes, missed that, which the tests told me about.